processing/src/main/java/org/apache/carbondata/processing/util/RemoveDictionaryUtil.java

```diff
@@ -123,6 +123,60 @@ private static int calculateTotalBytes(ByteBuffer[] byteBufferArr) {
   }
 
   /**
+   * This method will form one single byte [] for all the high card dims.
+   *
+   * @param byteBufferArr
+   * @return
+   */
```

Comment:
It is named like this long time back. We can remove it at the time of removing kettle based code.